index.js

```diff
@@ -690,10 +690,7 @@
   //. > S.min('10', '2')
   //. '10'
   //. ```
-  function min(x, y) {
-    return Z.lte(x, y) ? x : y;
-  }
-  S.min = def('min', {a: [Z.Ord]}, [a, a, a], min);
+  S.min = def('min', {a: [Z.Ord]}, [a, a, a], Z.min);
```

Comment:
Most Z-derived functions have a doc like:

> Curried version of `Z.fn`

Should we update all these docs?

Reply:
Which approach do you prefer, Aldwin? Linking to the `Z` functions or duplicating their descriptions? I see value in both, but I agree that we should be consistent.

Reply:
Duplication runs the risk of stale documentation in this project. That being said, I would prefer duplication. Every level of indirection is another annoyance.

Reply:
Can we build the documentation from Z's source?

Reply:
> Can we build the documentation from Z's source?

Potentially, but there's not a one-to-one correspondence between `Z` functions and their corresponding `S` functions. Consider [`Z.chainRec`][1] and [`S.chainRec`][2], for example.


[1]: https://github.com/sanctuary-js/sanctuary-type-classes#chainRec
[2]: https://sanctuary.js.org/#chainRec

Reply:
I've added a commit which adds descriptions for some of the Fantasy Land functions (taken from sanctuary-type-classes). Please have a look. :eyes:

Some functions still lack “proper” descriptions, but for a function such as [`S.sequence`][1] which is difficult to explain in words (without restating the type signature), I'm happy to link to [`Z.sequence`][2] which in turn links to [`fantasy-land/traverse`][3] (via [`Z.traverse`][4]).


[1]: https://sanctuary.js.org/#sequence
[2]: https://github.com/sanctuary-js/sanctuary-type-classes#sequence
[3]: https://github.com/fantasyland/fantasy-land#traverse-method
[4]: https://github.com/sanctuary-js/sanctuary-type-classes#traverse